app/views/general_list/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class=container-f>
+<h1 class="title-franck"> Shopping List </h1>
+<br>
+<div class="total">
+<h3>Total value of food needed : <%= @tot_price %></h3>
```

Comment:
Your total value of food just added unit prices. Kindly check your `@tot_price` in the GeneralListsController to see if it working properly. It's supposed to add all the value of food items